main.c

```diff
@@ -98,7 +98,7 @@ char *final_name = NULL, *asm_name = NULL, ext_incdir[MAX_NAME_LENGTH + 1];
 
 
 int main(int argc, char *argv[]) {
-  int parse_flags_result;
+  int parse_flags_result = FAILED;
```

Comment:
Already done in an another place, see https://github.com/vhelin/wla-dx/pull/217/commits/5f8309d4e893f5563fac4c651a20f6fd323b64f8

Reply:
Cool, line has been excluded.